dll/cpl/openglcfg/general.c

```diff
@@ -42,6 +33,12 @@ static VOID InitSettings(HWND hWndDlg)
     LoadString(hApplet, IDS_RENDERER_RSWR, (LPTSTR)szBultin, 127);
     SendDlgItemMessageW(hWndDlg, IDC_RENDERER, CB_ADDSTRING, 0, (LPARAM)szBultin);
 
+    if (RegCreateKeyExW(HKEY_CURRENT_USER, KEY_RENDERER, 0, NULL, 0, MAXIMUM_ALLOWED, NULL, &hKeyRenderer, NULL) != ERROR_SUCCESS)
+    {
+        RegCloseKey(hKeyDrivers);
```

Comment:
Closekey called before Open key (since bloc from original L19 is moved later in code) !!

Reply:
To be resolved ?

Reply:
I have added another commit if you mean that.
Or should the first one modified instead?